ubi8-py36/s2i_assemble.patch

```diff
@@ -1,47 +1,11 @@
-diff --git a/assemble b/assemble
-index c6515b5..3da074c 100755
---- a/assemble
-+++ b/assemble
-@@ -18,20 +18,6 @@ function virtualenv_bin() {
-   fi
- }
- 
--# Install pipenv to the separate virtualenv to isolate it
--# from system Python packages and packages in the main
--# virtualenv. Executable is simlinked into ~/.local/bin
--# to be accessible. This approach is inspired by pipsi
--# (pip script installer).
--function install_pipenv() {
--  echo "---> Installing pipenv packaging tool ..."
--  VENV_DIR=$HOME/.local/venvs/pipenv
--  virtualenv_bin "$VENV_DIR"
--  $VENV_DIR/bin/pip --isolated install -U pipenv
--  mkdir -p $HOME/.local/bin
--  ln -s $VENV_DIR/bin/pipenv $HOME/.local/bin/pipenv
--}
--
- set -e
- 
- shopt -s dotglob
-@@ -46,19 +32,8 @@ if [[ ! -z "$UPGRADE_PIP_TO_LATEST" ]]; then
-   pip install -U pip setuptools wheel
- fi
- 
--if [[ ! -z "$ENABLE_PIPENV" ]]; then
--  install_pipenv
--  echo "---> Installing dependencies via pipenv ..."
--  if [[ -f Pipfile ]]; then
--    pipenv install --deploy
--  elif [[ -f requirements.txt ]]; then
--    pipenv install -r requirements.txt
--  fi
--  # pipenv check
--elif [[ -f requirements.txt ]]; then
--  echo "---> Installing dependencies ..."
--  pip install -r requirements.txt
--fi
-+echo "---> Installing dependencies via micropipenv ..."
-+micropipenv install --deploy
- 
- if [[ -f setup.py && -z "$DISABLE_SETUP_PY_PROCESSING" ]]; then
-   echo "---> Installing application ..."
+--- a/assemble	2020-08-26 08:36:24.689696930 +0200
++++ b/assemble	2020-08-26 08:38:32.765236073 +0200
+@@ -1,4 +1,7 @@
+ #!/bin/bash
++# Thoth's addition always uses micropipenv.
```

Comment:
is the `+` required in these three lines.?

Reply:
Yes, it's a patch to assemble scripts.  The first `+` is from git patch, the second one is from the file itself.

Reply:
https://github.com/thoth-station/s2i-thoth/blob/951c03ffca9ce777118c6c42ad73a6a0ca491f20/ubi8-py36/Dockerfile#L28

Reply:
The build is successful locally for both ubi images.

Reply:
thanks @fridex